src/renderer/components/+preferences/kubernetes/helm-charts/activation-of-custom-helm-repository/custom-helm-repo.injectable.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import { observable } from "mobx";
+
+const customHelmRepoInjectable = getInjectable({
+  id: "custom-helm-repo",
+
+  instantiate: () => observable({
+    name: "",
+    url: "",
+    username: "",
+    password: "",
+    insecureSkipTlsVerify: false,
+    caFile: "",
+    keyFile: "",
+    certFile: "",
+  }),
+
+  lifecycle: lifecycleEnum.transient,
+});
+
+export default customHelmRepoInjectable;
```

Comment:
Why should this be an injectable and not just a function? When would we want to override this?

Reply:
No I mean, why not have the following function (yes I saw it is transient).

```ts
function newBlankCustomHelmRepo() {
  return observable({
    name: "",
    url: "",
    username: "",
    password: "",
    insecureSkipTlsVerify: false,
    caFile: "",
    keyFile: "",
    certFile: "",
  });
}
```

And then just call that?